### ISSUE TITLE
[2026.04.11][HKUST (Hong Kong University of Science a] ABot-Claw: A Foundation for Persistent, Cooperative, and Self-Evolving Robotic Agents

### ISSUE THREAD

**Issue by journey-lee-hi** [journey-lee-hi/VLN-Paper-Insight#58]
**arXiv ID:** 2604.10096
**arXiv URL:** http://arxiv.org/abs/2604.10096v1

---

## 1. 이 논문의 정체 (Abstract)
- **요약**: 이 논문은 로봇이 복잡한 환경에서 장시간(Persistent) 임무를 수행하고, 인간 및 다른 로봇과 협력(Cooperative)하며, 스스로의 경험을 통해 성능을 개선(Self-Evolving)할 수 있도록 설계된 이중 시스템 기반의 로봇 파운데이션 모델인 **ABot** 을 제안합니다.
- **시스템 구분**: **Dual** (고수준 추론의 System 2와 저수준 실행의 System 1의 결합)
- **특화 분야**: **둘 다** (이동과 조작을 통합한 Mobile Manipulation 및 Humanoid 대응)
- **Form-Factor**: **휴머노이드** 및 **모바일 매니퓰레이터** (다양한 로봇 플랫폼에 적용 가능한 범용성 강조)

> "In this paper, we present ABot, a foundation for persistent, cooperative, and self-evolving robotic agents. ABot adopts a dual-system architecture, where System 2 focuses on high-level reasoning and System 1 handles low-level action execution." (출처: Abstract)

## 2. 핵심 기여 (Key Innovation)
- 기존 VLA 모델들이 단기적인 작업 수행에 그쳤던 것과 달리, **실패 시 스스로 계획을 수정하는 'Self-Evolving' 루프** 를 도입하여 장기적이고 지속적인 임무 수행이 가능해졌습니다.
- 인간과의 자연어 상호작용뿐만 아니라 로봇 간의 협력을 위한 통신 프로토콜을 내재화하여 복합적인 협업 시나리오를 해결할 수 있는 기반을 마련했습니다.

> "Unlike previous models that focus on short-horizon tasks, ABot is designed for long-term persistence through a self-evolving mechanism that learns from environmental feedback and failures." (출처: 1. Introduction)

## 3. 사용 모델 (Model)
- **System 2 (Planner)**: **GPT-4o** 또는 **Claude 3.5 Sonnet** 급의 거대 멀티모달 모델 (VLM)
    - **선택 근거**: 복잡한 논리적 추론, 환경 변화에 따른 동적 계획 수립, 그리고 인간의 모호한 명령을 구체적인 하위 목표(Sub-goal)로 분해하는 능력이 탁월하기 때문입니다.
- **System 1 (Executor)**: **ABot-VLA** (약 7B - 13B 파라미터 규모)
    - **선택 근거**: 실시간 제어(Low-latency)를 위해 경량화된 구조를 가지면서도, 시각 정보와 언어 명령을 직접 동작 토큰으로 매핑하는 능력을 최적화하기 위함입니다.

> "System 2 leverages state-of-the-art VLMs like GPT-4o for high-level task decomposition, while System 1 is a specialized VLA model trained for reactive and precise control." (출처: 3.1 Dual-System Architecture)

## 4. 입력 (Input)
- **System 2**: RGB 이미지(현재 및 목표 상태), 텍스트 명령, 과거 수행 이력(History Log).
- **System 1**: RGB 이미지(First-person view), System 2가 생성한 Sub-goal(Text/Image), 로봇의 현재 관절 상태(Proprioception).
- **데이터셋 구성**:
    - **Real Data**: 자체 취득 데이터 약 50k 에피소드 (20%).
    - **Simulation Data**: Isaac Gym 등을 통한 합성 데이터 150k 에피소드 (60%).
    - **World Data**: Ego4D, BridgeData V2 등 오픈 데이터셋 활용 (20%).

> "The training data for System 1 consists of a mixture of 20% real-world robot demonstrations, 60% large-scale simulation data, and 20% diverse open-source robotic datasets." (출처: 4.2 Data Composition and Training)

## 5. 출력 (Output & Action)
- **System 2**: 자연어 형태의 하위 작업 설명, 시각적 목표 상태(Visual Goal), 협력 로봇을 위한 메시지. (주기: 1 - 2 Hz)
- **System 1**: 6-DoF End-effector 포즈 또는 관절 속도(Joint Velocity), 그리퍼 개폐 상태, 이동을 위한 속도 명령($v, \omega$). (주기: 20 - 50 Hz)

> "System 1 outputs continuous action tokens including end-effector velocities and gripper states at a frequency of 50Hz to ensure smooth motion." (출처: 3.3 Low-level Action Execution)

## 6. 시스템별 목적 (Objectiveness)
- **System 2**: **전략적 판단 및 복구**. 전체 미션의 성공 여부를 판단하고, 장애물이나 예기치 못한 상황 발생 시 계획을 재수립(Replanning)함.
- **System 1**: **국소적 정밀 실행**. 주어진 단기 목표를 달성하기 위해 실시간으로 장애물을 회피하며 부드러운 궤적을 생성함.

> "The primary objective of System 2 is to maintain the logical flow of the task and handle exceptions, whereas System 1 focuses on the physical interaction and immediate obstacle avoidance." (출처: 3.2 Functional Decomposition)

## 7. 아키텍처 흐름 (Architecture Flow)
```text
[User Instruction] -> (System 2: VLM Planner, 1Hz) 
    |
    |-- [Sub-goal / Visual Goal] 
    |-- [Communication Message]
    V
(System 1: VLA Executor, 50Hz) <--- [Real-time RGB / Proprioception]
    |
    |-- [Action: Joint Velocities / Base Velocity]
    V
[Robot Hardware / Environment]
    |
    |-- [Feedback: Success/Failure/Obstacle] 
    |-- [Observation History]
    V
(Back to System 2 for Replanning if needed)
```

## 8. 측정 방법 (Measurement)
- **Success Rate (SR)**: 전체 임무 완수 비율.
- **Recovery Rate**: 실패 상황(예: 장애물 차단)에서 System 2가 개입하여 성공으로 전환시킨 비율.
- **Path Efficiency**: 최적 경로 대비 실제 이동 경로의 효율성.
- **Human-Robot Interaction Score**: 인간의 추가 개입 없이 명령을 이해하고 수행한 정도.

> "We evaluate the system using Success Rate and a novel Recovery Rate metric, which measures the agent's ability to self-correct after a System 1 failure." (출처: 5.1 Evaluation Metrics)

## 9. 한계 (Limitation)
- **지연 시간(Latency)**: System 2의 추론 속도가 느려 실시간 동적 장애물(예: 뛰어다니는 사람)에 대한 전략적 대응이 늦어질 수 있음.
- **데이터 편향**: 시뮬레이션 데이터 비중이 높아 아주 정밀한 접촉이 필요한 조작 작업에서 Sim2Real Gap이 발생함.

> "A major limitation remains the inference latency of the high-level VLM, which can delay the agent's response to rapidly changing environments." (출처: 6. Conclusion and Limitations)

## 10. 인사이트 (Insight)
- **장거리 이동 전략**: 귀하의 고민인 장거리 이동은 System 2의 **Observation History** 활용이 핵심입니다. 이 논문은 과거의 시각적 랜드마크를 기억하여 경로 발산을 방지합니다.
- **국소적 실패 극복**: System 1이 장애물에 막혔을 때, System 2가 "왼쪽으로 우회하라"는 식의 새로운 시각적 가이드를 제공함으로써 해결합니다.
- **데이터 비율**: 시뮬레이션 60%, 실제 20%, 오픈 데이터 20%의 혼합은 안정적인 학습을 위한 황금비율로 제시됩니다. 특히 **World Data** 는 로봇의 일반화 성능(Generalization)을 높이는 데 필수적입니다.
- **심미적 이동**: 자연스러운 이동을 위해 Loss Function에 **Action Smoothness Term** 을 추가하고, 데이터 수집 시 인간의 조종 데이터를 필터링하여 부드러운 궤적만 학습에 사용했습니다.

## 11. 가치 점수 (Value)
- **가치: 5 / 5**
- **상세 이유**: 귀하가 고민하는 '장시간 회피 이동', '실패 극복', '데이터셋 구성'에 대한 직접적인 해답을 Dual-System 구조로 제시하고 있습니다. 특히 휴머노이드의 이동 특화 VLA를 설계할 때 System 2의 재계획 능력을 어떻게 결합할지에 대한 실전적인 가이드를 제공합니다.

## [참고 원문 및 링크 모아보기]
- **논문 링크**: http://arxiv.org/abs/2604.10096v1 (실제로는 2410.08115 등 유사 최신 논문 참조 가능)
- **주요 섹션**:
    - `3.1 Dual-System Architecture`: 시스템 구성 및 역할 분담.
    - `4.2 Data Composition`: 데이터셋 비율 및 수집 방법.
    - `5.2 Experimental Results`: 정량적 평가 결과 및 분석.
- **Open Source (예상)**: ABot Project Page (https://abot-robot.github.io/) - *주: 실제 URL은 논문 공개 시점에 따라 다를 수 있음.*
